packages/drivers/basewebdriver/index.ts

```diff
@@ -0,0 +1,91 @@
+import WebTestingFrameworkDriverHelper from "@wix-pilot/web-utils";
+import type { ElementMatchingCriteria, Page } from "@wix-pilot/web-utils";
+import type {
+  TestingFrameworkAPICatalog,
+  TestingFrameworkDriver,
+  TestingFrameworkDriverConfig,
+} from "@wix-pilot/core";
+import { baseDriverCategories } from "./baseDriverCategories";
+import extendAPICategories from "./extendAPICategories";
+
+export class BaseWebDriver<T extends Page> implements TestingFrameworkDriver {
+  private driverUtils: WebTestingFrameworkDriverHelper;
+
+  constructor(
+    private frameworkApiCatalog: TestingFrameworkAPICatalog,
```

Comment:
don't think we need this param here, better to define it on the base driver for all